specification/compute/resource-manager/Microsoft.Compute/stable/2020-12-01/examples/UpdateAManagedDiskToAddSupportsHibernation.json

```diff
@@ -0,0 +1,57 @@
+{
+  "parameters": {
+    "subscriptionId": "{subscription-id}",
+    "resourceGroupName": "myResourceGroup",
+    "api-version": "2020-12-01",
+    "diskName": "myDisk",
+    "disk": {
+      "properties": {
+        "supportsHibernation": true
+      }
+    }
+  },
+  "responses": {
+    "202": {
+      "body": {
+        "name": "myDisk",
+        "location": "westus",
+        "sku": {
+          "name": "Standard_LRS",
+          "tier": "Standard"
+        },
+        "properties": {
```

Comment:
Does PATCH api return tags as well? It should if not.

Reply:
PATCH api does return tags. However the examples in question dont have tags in the request or the resource, so tags are not present in the response object.